diamond.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+
+int main(){
+	int i,j,k;
+	k=10-i;
+	for(i=1;i<=5;i++){
+		for(j=1;j<=i;j++){
+		printf("*");
+		continue;
+		}
+		if(j=i+1){
```

Comment:
Hello!

Actually I don't really think that you understand my code about printing diamond. I am just setting a range for the j, and j is the number of row. 

By the way I have fixed all my code with the teacher this afternoon, should I push it again?

Sorry for disturbing, and thank you!

Best wishes!

Sheqi Zhang 

发自我的 iPhone

> 在 2016年7月1日，下午10:52，Lesley Cordero <notifications@github.com> 写道：
> 
> In diamond.c:
> 
> > @@ -0,0 +1,26 @@
> > +#include <stdio.h>
> > +
> > +int main(){
> > +	int i,j,k;
> > +	k=10-i;
> > +	for(i=1;i<=5;i++){
> > +		for(j=1;j<=i;j++){
> > +		printf("*");
> > +		continue;
> > +		}
> > +		if(j=i+1){
> this should be j==i+1 because it's comparing and not assigning.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>